tests/Behat/Context/GroupsContext.php

```diff
@@ -115,4 +149,72 @@ public function thisAttributesShouldBeVisibleNextToTheGroup(string $group, Table
             $this->indexPage->areAttributesVisible($group, $attributes)
         );
     }
+
+    /**
+     * @Given the store has a product attribute group :groupName with attributes:
+     */
+    public function theStoreHasAProductAttributeGroup(string $groupName, TableNode $table)
+    {
+        $group = new Group();
+        $group->setName($groupName);
+
+        $attributeCodes = array_merge([], ...$table->getRows());
+
+        foreach ($attributeCodes as $attributeCode) {
+            $syliusAttribute = new ProductAttribute();
+            $syliusAttribute->setCode($attributeCode);
+            $syliusAttribute->setType('text');
+            $syliusAttribute->setStorageType('text');
+            $syliusAttribute->setTranslatable(false);
+
+            $attribute = new Attribute();
+            $attribute->setGroup($group);
+            $attribute->setSyliusAttribute($syliusAttribute);
```

Comment:
you can move this logic to factories and after that just use repostiory->add($object). It is only sugestion. if it's a lot of work just leave it.

Reply:
I think it will take us extra time that we no longer have